test/studies/prk/stencil/PERFCOMPOPTS

```diff
@@ -0,0 +1 @@
+-sassertNoSlicing
```

Comment:
I suggested using -sassertNoSlicing as a means of understanding why our performance was 2x off from the reference versions, but agree that our preference would be not to use it in performance testing.  Or, I could imagine running the test in two configurations (w/ and w/out assertNoSlicing) and plotting both.  But down the road, the intention is to retire this flag which might require some housekeeping on the .dat files at that point.